tests/greenheart/test_hydrogen/test_greenheart_system.py

```diff
@@ -95,175 +97,152 @@ def test_simulation_wind_wave(subtests):
         output_level=5,
     )
 
+    lcoe, lcoh, _, hi = run_simulation(config)
+
     # TODO base this test value on something
     with subtests.test("lcoh"):
-        assert lcoh == approx(8.120065296802442)
+        assert lcoh == approx(8.133894926897252)
 
     # prior to 20240207 value was approx(0.11051228251811765) # TODO base this test value on something
     with subtests.test("lcoe"):
-        assert lcoe == approx(0.12863386719193057)
-
-    with subtests.test("energy sources"):
-        expected_annual_energy_hybrid = (
-            hi.system.annual_energies.wind + hi.system.annual_energies.wave
-        )
-        assert hi.system.annual_energies.hybrid == approx(expected_annual_energy_hybrid)
+        assert lcoe == approx(0.12887769358919945)
 
 
 def test_simulation_wind_wave_solar(subtests):
     filename_hopp_config_wind_wave_solar = os.path.join(
         orbit_library_path, f"plant/hopp_config_wind_wave_solar.yaml"
     )
 
-    lcoe, lcoh, _, hi = run_simulation(
-        filename_hopp_config_wind_wave_solar,
-        filename_greenheart_config,
-        filename_turbine_config,
-        filename_orbit_config,
-        filename_floris_config,
+    config = GreenHeartSimulationConfig(
+        filename_hopp_config=filename_hopp_config_wind_wave_solar,
+        filename_greenheart_config=filename_greenheart_config,
+        filename_turbine_config=filename_turbine_config,
+        filename_orbit_config=filename_orbit_config,
+        filename_floris_config=filename_floris_config,
         verbose=False,
         show_plots=False,
         save_plots=False,
         use_profast=True,
-        post_processing=False,
+        post_processing=True,
         incentive_option=1,
-        plant_design_scenario=9,
+        plant_design_scenario=7,
```

Comment:
why the change in design scenario? Is this accurate? Seems like the possible cause for the large change in test output value

Reply:
I've addressed this by adding a new design scenario (11). The test value for lcoh is now very close to what it was before (12.59)